### PR TITLE
Newsletter 12: Add Robo, 4K Intro,  A\B Street, Protochess

### PR DESCRIPTION
**Pull request by ozkriff**
Hey, @janiorca @alexheretic @dabreegster @raytran, should something be changed or added? r?

_Part of #201 ("Newsletter 12: Coordination/Tracking")_

### CODE REVIEW

File: content/posts/newsletter-012/index.md [hunk 1]
```diff
@@ -62,6 +62,59 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+[![Youtube preview: mountains & spheres](4k-into-youtube.jpeg)][4k-video]
+
+^ _Click to [watch the demo on Youtube][4k-video]._
+
+Jani Peltonen has recently released a [4K intro][4k-src]
+which is completely written in Rust and GLSL
+and published an article ["Writing a winning 4K intro in Rust"][4k-post]:
+
+> A 4K intro is a demo where the entire program (including any data)
+> has two be 4096 bytes or less so it is important
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-012/index.md:76]: Presumably this should be 'has to be', but that's a mistake from the original post, so might be fine to leave it in.

File: content/posts/newsletter-012/index.md [hunk 2]
```diff
@@ -144,6 +197,27 @@ _Discussions:
...
+how small changes to roads affect cyclists, transit users, pedestrians,
+and drivers.
+
+This month [versions v0.2.2..v0.2.5][abstreet-releases] were released.
+Some of the updates:
+
+- New random traffic scenario generator that makes people go between
+  houses and workplaces.
+- New commute pattern explorer tool.
+- New character art to give cutscenes a bit more personaliy.
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-012/index.md:214]:
```suggestion
- New character art to give cutscenes a bit more personality.
```

File: content/posts/newsletter-012/index.md [hunk 3]
```diff
@@ -162,6 +236,28 @@ This release features:
 [tennis-academy-dash]: https://iolivia.itch.io/tennis-academy-dash
 [tennis-academy-update]: https://twitter.com/oliviff/status/1285298082033348609
 
+### [protochess]
+
+[![demo](protochess.gif)][protochess]
+
+[protochess] ([source][protochess-src]) in an online multiplayer chess website
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-012/index.md:243]:
```suggestion
[protochess] ([source][protochess-src]) is an online multiplayer chess website
```

**Comment by ozkriff** [comment]
:timer_clock: Merging in a few hours if there're no objections.

**Comment by dabreegster** [comment]
LGTM for A/B Street. I've been swamped this week, didn't get around to this; thanks for the write-up!

**Comment by raytran** [comment]
Protochess looks good 👌 thanks for doing this

**Pull request merged**